### PR TITLE
Update peaks2genes

### PR DESCRIPTION
**Pull request by cmonjeau**

### CODE REVIEW

**Comment by cmonjeau** [comment]
Info : for this tutorial, the dataset mm9.RefSeq_genes_from_UCSC used in the part 2 appears to be missing from the shared data.

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md [hunk 1]
```diff
@@ -68,19 +68,11 @@ Let's start with a fresh history.
 
 > ### {% icon hands_on %} Hands-on: Data upload
 >
-> 1. Download the list of peak regions (the file [`GSE37268_mof3.out.hpeak.txt.gz`](https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE37268&format=file&file=GSE37268%5Fmof3%2Eout%2Ehpeak%2Etxt%2Egz)) from [GEO](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37268) to your computer 
+> 1. Download the list of peak regions (the file [`GSE37268_mof3.out.hpeak.txt.gz`](https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE37268&format=file&file=GSE37268%5Fmof3%2Eout%2Ehpeak%2Etxt%2Egz)) from [GEO](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37268) to your computer
 > 2. Click on the upload button in the upper left ot the interface
 >
 >    ![Upload icon](../../images/upload_button.png)
 >
->    > ### {% icon tip %} Tip: Importing data via links
```
Comment (bgruening) [inline topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: Why have you removed this tip?

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md [hunk 2]
```diff
@@ -482,11 +474,12 @@ We need to generate a new BED file from the original peak file that contains the
 Now we cut out just the chromosome plus the start and end of the summit:
 
 > ### {% icon hands_on %} Hands-on: Cut out columns
-> 1. **Cut columns from a table** {% icon tool %} with the following settings:
+> 1. **Cut** {% icon tool %}: Run the **Cut columns from a table** with the following settings:
```
Comment (nsoranzo) [inline topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: `Run the Cut`: remove `the`

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md (hunk 1 above)
Comment (cmonjeau) [reply topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: Tip is duplicated after step 5

File: topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md [hunk 3]
```diff
@@ -218,11 +211,9 @@ As we directly see, the peak file lacks `chr` before any chromosome number. But
...
 > ### {% icon hands_on %} Hands-on: View end of file
 >
-> 1. Search for the **Select last** {% icon tool %} tool in the tool panel on the left
-> 2. Select the following settings
->     - **Text file** to our peak file `GSE37268_mof3.out.hpeak.txt`
->     - **Operation**: `Keep last lines`
->     - **Number of lines**: Choose a value, e.g. `100`
+> 1. **Select last** {% icon tool %}: Run **Select last lines from a dataset** with the following settings:
+>     - **Select last**: Choose a value, e.g. `100`
+>     - **from**: Our peak file `GSE37268_mof3.out.hpeak.txt`
```
Comment (nsoranzo) [inline topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md]: That's not correct, this tutorial is using the tool [tp_tail_tool](https://github.com/bgruening/galaxytools/blob/master/tools/text_processing/text_processing/tail.xml) ("Select last lines from a dataset (tail)") from the [text_processing](https://toolshed.g2.bx.psu.edu/view/bgruening/text_processing/) ToolShed repository, not the tool [Show tail1](https://github.com/galaxyproject/galaxy/blob/dev/tools/filters/tailWrapper.xml) ("Select last lines from a dataset") coming together with Galaxy.
See https://github.com/galaxyproject/training-material/blob/master/topics/introduction/tutorials/galaxy-intro-peaks2genes/tools.yaml#L11

**Pull request merged**